sem/HW/hw_4/asince.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+import aiohttp
+import time
+from PIL import Image
+
+
+async def download(url):
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url) as response:
+            img = response.content
+            filename = 'asyncio_' + url.split('/')[-1]
+            with open(f"img/{filename}", "wb") as f:
```

Comment:
Чтобы работало на всех платформах, можно сделать так
```suggestion
            with open(os.path.join(img, filename), "wb") as f:
```

Reply:
И папки `img` у вас по умолчанию нет. Поэтому лучше бы проверять на ее на существование и создавать, если ее нет